### DISCUSSION TITLE
几个位置设置aria2的地址，各是什么作用？

### DISCUSSION THREAD

**Discussion by crarm** [AlistGo/alist#2305] [General]
![image](https://user-images.githubusercontent.com/61692711/201461253-ce710ceb-8278-4b38-afb4-23dfa2248216.png)
这里的设置是后台下载地址吗？

![image](https://user-images.githubusercontent.com/61692711/201461288-07d08bca-2120-4f11-a1fe-8a40bbbfa9d9.png)
这是发送到哪里？是发送到上面的还是发送到下面的？

![image](https://user-images.githubusercontent.com/61692711/201461320-cae8c716-47c0-4ce2-b1d1-fca90725bde0.png)
这个又应该设什么地址？

**Comment by xhofe** [comment]
- 图1后台管理中设置的Aria2：用于离线下载，即给定链接和路径后会使用这里的aria2进行下载，完成后上传的路径对应的存储。这里的aria2需要是安装alist的机器可以访问的。
- 图2发送到Aria2即发送到图3中设置的aria2，一般用于本地下载，即下载选中的文件到自己的电脑。这里的aria2不需要安装alist的机器可以访问。

**Reply by crarm** [reply]
谢谢，尝试了一下，明白了

**Reply by crarm** [reply]
感觉页面可以写的清楚一点，"发送到aria2"可以改写为"发送到本地aria2”下载，
![image](https://user-images.githubusercontent.com/61692711/201474093-e5b8ab4d-d973-48b2-b4e8-285edfd22872.png)
此处离线下载可以改到“发送到aria2”一起，且改为“发送到远程aria2”类似的好区分的语句
两个下载在一起更方便一点吧

**Reply by crarm** [reply]
![image](https://user-images.githubusercontent.com/61692711/201474225-5d4f4f0c-dfef-4e57-ab82-6b6c99ca794d.png)
离线下载到tmp目录成功了，可是转移不成功，应该如何设置
看图片，如果我在天翼盘的下载链接在115盘目录进行离线下载，alist自身会将链接再上传的115的目录，可以不采用这一步吗，只下载到tmp目录即停止？

**Reply by crarm** [reply]
另外，使用tmp来设置缓存目录的目的是什么呢？
直接使用aria2的默认下载目录是不是更方便，更简洁？如果alist、aria2都是容器，这个目录绕来绕去的有点复杂

**Reply by qinghuannn** [reply]
> ![image](https://user-images.githubusercontent.com/61692711/201474225-5d4f4f0c-dfef-4e57-ab82-6b6c99ca794d.png) 离线下载到tmp目录成功了，可是转移不成功，应该如何设置 看图片，如果我在天翼盘的下载链接在115盘目录进行离线下载，alist自身会将链接再上传的115的目录，可以不采用这一步吗，只下载到tmp目录即停止？

看到解决办法了，是aria2的路径和alist路径映射要一致。具体的可以在discussion中搜一下
--update
遇到相同问题，但目前没看到相关解决办法。
